jobs/pulse3d/manifests/base/job-runner.yaml

```diff
@@ -7,5 +7,9 @@ spec:
   job_queue: pulse3d
   seconds_to_poll_db: 15
   max_num_of_workers: 8
+  ecr_repo: null
+  product_specific:
+    pulse3d_uploads_bucket: null
```

Comment:
Does this need to be product specific? Since each product will have it's own s3 bucket for data/logs it seems like this could be `uploads_bucket` and `analyzed_bucket` and the queue processor will grab those and pass them to the jobs

Reply:
@jperson  Yeah we can do that. I didn't know what future products would need so was just leaving it open ended to add whichever and however many, but happy to change it if you think we'll always need the same variables for each product.